app/models/case_attachment.rb

```diff
@@ -123,11 +123,11 @@ def remove_from_storage_bucket
   def validate_file_extension
     mime_type = Rack::Mime.mime_type(File.extname filename)
     unless Settings.case_uploads_accepted_types.include? mime_type
-      errors[:url] << I18n.t(
+      errors.add(:url, I18n.t(
```

Comment:
Is this just some housekeeping? Or is this related to the filters in some way I cant understand? 🤔  Either way 👍